src/transformState.js

```diff
@@ -5,7 +5,33 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (let i = 0; i < actions.length; i++) {
```

Comment:
Consider using a 'for of' loop instead of a 'for' loop for better readability (const action of actions)